APISpecification.yaml

```diff
@@ -67,11 +73,23 @@ paths:
             default: a7335667-93e7-11ec-a39d-005056b38ce3
       responses:
         '200':
-          description: successful operation
+          description: Successful operation
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenomeDetails'
+        '404':
+          description: Genome not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Could not find details for a7335667-93e7-11ec-a39d-005056b38ce3"
```

Comment:
Again: this is not my code so I don't know what was the itention of the implementer. I just added the message from the code to API spec because the spec should reflect the reality. Improving error messages across all endpoints is out of scope of this PR (as was probably updating the spec but it's part of maintenance)